auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -532,6 +489,13 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
 <style lang="scss" scoped>
   @import '$assets/scss/theme.scss';
 
+  .loading-container.grayed-out {
+    // these are the same styles as dialog overlay:
+    opacity: 0.46;
+    background-color: rgb(33, 33, 33); // grey darken-4
+    border-color: rgb(33, 33, 33); // grey darken-4
```

Comment:
This changes the spinner to semi-transparent, so the user can see that the page is loading. It makes this "feel" faster :)